Check/Views/CacheOutput.php

```diff
@@ -0,0 +1,193 @@
+<?php
+/**
+ * @file
+ * Contains \SiteAudit\Check\Views\CacheOutput.
+ */
+
+/**
+ * Class SiteAuditCheckViewsCacheOutput.
+ */
+class SiteAuditCheckViewsCacheOutput extends SiteAuditCheckAbstract {
+  /**
+   * Implements \SiteAudit\Check\Abstract\getLabel().
+   */
+  public function getLabel() {
+    return dt('Rendered output caching');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getDescription().
+   */
+  public function getDescription() {
+    return dt('Check the length of time raw rendered output should be cached.');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultFail().
+   */
+  public function getResultFail() {
+    return dt('No View is caching rendered output!');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultInfo().
+   */
+  public function getResultInfo() {
+    return $this->getResultWarn();
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultPass().
+   */
+  public function getResultPass() {
+    return dt('Every View is caching rendered output.');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultWarn().
+   */
+  public function getResultWarn() {
+    return dt('The following Views are not caching rendered output: @views_without_output_caching', array(
+      '@views_without_output_caching' => implode(', ', $this->registry['views_without_output_caching']),
+    ));
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getAction().
+   */
+  public function getAction() {
+    if (!in_array($this->score, array(SiteAuditCheckAbstract::AUDIT_CHECK_SCORE_INFO, SiteAuditCheckAbstract::AUDIT_CHECK_SCORE_PASS))) {
+      $ret_val = dt('Rendered output should be cached for as long as possible (if the query changes, the output will be refreshed).');
```

Comment:
Tag caching for each view is not automatically done and has to be enabled by the user. There is an option between time based caching and Tag based caching. So this statement should be correct.